lib/find.js

```diff
@@ -30,7 +30,7 @@ function pointFinder(name) {
 
             return data.Points.filter(function (p) {
                 p = p.split('|');
-                return p[2].length === 0 || p[2] === 'Dresden';
+                return p[3] && (p[2].length === 0 || p[2] === 'Dresden');
```

Comment:
why do you check p[3]? this is the stop name, or? are there stops without a name?